File: series3.c
#include<stdio.h>
int main()
{
    int n,i=1,sum=0,f=1;
    printf("Enter the number: \n");
    scanf("%d",&n);
    while(i<=n)
    {
            f=f*i;
        printf("%d ",f);
        sum=sum+i*i;
        i++;
    }
    printf("\nsum is: %d\n",sum);
    return 0;
}

File: Series3.java
import java.util.*;
class  Series3
{
    public static void main(String[]args)
    {
        Scanner sc=new Scanner(System.in);
        int n,i,s,sum=0,j;
        System.out.println("Enter the number: ");
        n=sc.nextInt();
        for(i=1;i<=n;i++){
                s=0;
            for(j=1;j<=i;j++)
                sum=sum+j*j;
            }
            System.out.println(sum);
    }
}
